common/defaults.go

```diff
@@ -14,66 +14,16 @@
 
 package common
 
+// app-controller
 const (
 	// ArgoCDApplicationControllerComponent is the name of the application controller control plane component
 	ArgoCDApplicationControllerComponent = "argocd-application-controller"
 
 	// ArgoCDApplicationControllerDefaultShardReplicas is the default number of replicas that the ArgoCD Application Controller Should Use
 	ArgocdApplicationControllerDefaultReplicas = 1
```

Comment:
Should this key be like `ArgoCD...` instead of `Argocd...` as other keys?

Reply:
nice catch

Reply:
Although, this will be changing soon in my app-controller PR when all these constants will be moved and renamed in the app-controller package